src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -89,17 +89,41 @@ std::vector<NodeEntry> OfflineParams(std::vector<NodeEntry>&& outputs,
   return outputs;
 }
 
-inline bool NeedQuantize(NodePtr node, const std::unordered_set<NodePtr> excluded_nodes) {
+inline bool NeedQuantize(NodePtr node, const std::unordered_set<std::string>& excluded_nodes) {
   static auto& quantized_op_map = Op::GetAttr<mxnet::FQuantizedOp>("FQuantizedOp");
-  return quantized_op_map.count(node->op()) && !excluded_nodes.count(node);
+  if (quantized_op_map.count(node->op())) {
+    bool excluded = false;
+    if (node->attrs.subgraphs.size()) {
+      // This is a subgraph node, try to match subgraph name first,
+      // and then try to match inner node.
+      if (excluded_nodes.count(node->attrs.name)) {
+        excluded = true;
+      } else {
+        auto subgraph_sym = node->attrs.subgraphs[0];
```

Comment:
Please add a comment explaining what index 0 correspond to. If possible, add a CHEKC_EQ as well for `node->attrs.subgraphs`.

Reply:
OK. But this is not a good design that we didn't have a rule that which index should be the subgraph symbol node as I comment in your subgraph PR. We have to assume that the first node(index 0) is the subgraph node, and all developers won't break this.

Reply:
why are there more than 1 subgraph on a subgraph node?

Reply:
You have the full control of creating subgraph(s) in your subgraph property implementation. As we discussed before, if you plan to have more subgraphs, it's better to use enum as indices which are more descriptive.

Reply:
the `subgraphs` field isn't only used by the subgraph API. control flow operators also use this field to store subgraphs.

Reply:
CHECK is added.